modules/ROOT/pages/app-versioning.adoc

```diff
@@ -74,17 +74,17 @@ This model means there is zero downtime while updates are applied to the applica
 [NOTE]
 Since updates are applied as rolling updates, there will be different application versions running at the same time until the update is completed, so make sure your applications are able to run multiple versions in parallel.
 +
-image::app-versioning-replica-status.png[]
+image::app-versioning-replica-status.png["Status of replicas for a Mule app in the Runtime Manager UI"]
```

Comment:
```suggestion
image::app-versioning-replica-status.png["Replicas status for a Mule app in the Runtime Manager UI"]
```
Or you can use "Status of replicas" in all other PR entries to match the content.